README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
     basic_auth:
       # Corresponds to your OpenStack User and Project
-      username: <user_id>@<project_id>
+      username: <user_id>|<project_id>
```

Comment:
I wasn't arguing on using names rather than ids. You can scope to projects as well as domains. For that the `@` feels more natural to me than the `|` character. So it would be`<username>@<domain_name>`, `<username>@<project_id>`. If you want to use the project's name maybe it could be `<username>@<project_name>|<domain_name>`.
Or if the user is in a different domain `<username>|<domain_name>@<project_name>|<domain_name>`.  The `@` would always qualify the scope.